BookEcommerce.Services/CartService.cs

```diff
@@ -0,0 +1,117 @@
+﻿using BookEcommerce.Models.DAL.Interfaces;
+using BookEcommerce.Models.DTOs;
+using BookEcommerce.Models.DTOs.Request;
+using BookEcommerce.Models.DTOs.Response;
+using BookEcommerce.Models.Entities;
+using BookEcommerce.Services.Base;
+using BookEcommerce.Services.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookEcommerce.Services
+{
+    public class CartService : BaseService, ICartService
+    {
+        private readonly ICartRepository cartRepository;
+        private readonly IProductVariantRepository productVariantRepository;
+        private readonly IProductPriceRepository productPriceRepository;
+        public CartService(IUnitOfWork unitOfWork, ICartRepository cartRepository, 
+                           IProductVariantRepository productVariantRepository, IProductPriceRepository productPriceRepository) : base(unitOfWork)
+        {
+            this.productVariantRepository = productVariantRepository;
+            this.cartRepository = cartRepository;
+            this.productPriceRepository = productPriceRepository;
+        }
+        public async Task<CartResponse> AddCart(CartRequest req, string cusId)
+        {
+            try
+            {
+                var findCart = await cartRepository.GetCart(cusId);
+                foreach (var product in findCart)
+                {
```

Comment:
get Cart but actually it's products list. So for me, it should be a property in cart like Cart.Items